app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age_list",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
+    ]
+)
+def test_should_except_values(
+        cat_age: int,
+        dog_age: int,
+        human_age_list: int
```

Comment:
The type annotation for `human_age_list` should be `List[int]` instead of `int` because it represents a list of integers. Make sure to import `List` from `typing` if you decide to use it.